Project.toml

```diff
@@ -14,7 +14,7 @@ Tables = "bd369af6-aec1-5ad0-b16a-f7cc5008161c"
 
 [compat]
 AxisArrays = "0.4"
-Compat = "3.3"
+Compat = "2.2, 3"
```

Comment:
OTOH - do we really need Compat anymore? We only need it for `eachslice ` (which I should add as an import)